src/lib/__tests__/Preview-test.js

```diff
@@ -795,6 +795,34 @@ describe('lib/Preview', () => {
         });
     });
 
+    describe('canPrint()', () => {
+        beforeEach(() => {
+            stubs.canDownload = sandbox.stub(file, 'canDownload');
+            stubs.checkFeature = sandbox.stub(file, 'checkFeature');
+        });
+
+        it('should return true is file is downloadable and has printing feature', () => {
+            stubs.canDownload.returns(true);
+            stubs.checkFeature.withArgs(sinon.match.any, 'print').returns(true);
```

Comment:
Try to avoid using withArgs if possible. It’s difficult to migrate to other test frameworks like Jest.